client/my-sites/migrate/section-migrate.jsx

```diff
@@ -472,6 +472,11 @@ class SectionMigrate extends Component {
 					{ this.renderStartTime() }
 					{ this.renderProgressBar() }
 					{ this.renderProgressList() }
+					<p class="migrate__note">
+						{ translate(
+							"You can safely navigate away from this page if you need to; we'll send you a notification when it's done"
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'You can safely navigate away from this page if you need to; we'll send you a notification when it's done.' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/migrate-restoring-content)

Reply:
@bogiii could you add a full-stop so that the translation can be re-used?

Reply:
Yes, sure. Done.